tests/robustness/report/client.go

```diff
@@ -169,36 +174,36 @@ func loadKeyValueOperations(path string) (operations []porcupine.Operation, err
 	return operations, nil
 }
 
-func persistWatchOperations(t *testing.T, lg *zap.Logger, path string, responses []model.WatchOperation) {
+func persistWatchOperations(lg *zap.Logger, path string, responses []model.WatchOperation) error {
 	lg.Info("Saving watch operations", zap.String("path", path))
 	file, err := os.OpenFile(path, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0o755)
 	if err != nil {
-		t.Errorf("Failed to save watch operations: %v", err)
-		return
+		return fmt.Errorf("Failed to save watch operations: %v", err)
```

Comment:
This makes sense, however we need to lowercase the error message as the static validation will requires it.